enclave/src/lib.rs

```diff
@@ -359,6 +359,8 @@ pub unsafe extern "C" fn init_chain_relay(
     // info!(" Polkadex Gateway Nonces and Cache Initialized");
 
     nonce_handler::create_in_memory_nonce_storage(); //FIXME Error handling required
+    polkadex_balance_storage::create_in_memory_balance_storage();
+    polkadex_orderbook_storage::create_in_memory_orderbook_storage(vec![]);
```

Comment:
This is already done here: 
https://github.com/Polkadex-Substrate/polkadexTEE-worker/blob/develop/worker/src/main.rs#L547
respective in the enclave:
https://github.com/Polkadex-Substrate/polkadexTEE-worker/blob/develop/enclave/src/lib.rs#L403

Reply:
done